### ISSUE TITLE
(x, y ) cordinate is relative to grid or relative to whole image.

### ISSUE THREAD

**Issue by vikrant094** [experiencor/basic-yolo-keras#201]
Adjust prediction
     @experiencor 
    ### adjust x and y      
    pred_box_xy = tf.sigmoid(y_pred[..., :2]) + **cell_grid**

in above line (x, y ) coordinate is relative to grid or relative to whole image.
and what is actual  meaning of true_boxes.
in my case i have training set instance  of dimension (13,13,5,85). let say..
where,
 (13,13) grid , 5 is no of anchor_box., and out of 85 , 80 is classes other is bx,by,bh,bw, pc(object probability) right????
 then how can u infer true_boxes from it. any help will be appreciated..

**Comment by voqtuyen** [comment]
The coordinates are calculated relatively to the grid cell, the images is divided into 13x13 grid and the top left corner is (0,0), the bottom right corner is (13x13). The outputs x,y of CNN have to be between 0 and 1, thus we apply sigmoid to make sure that the center/midpoint of an object always lies within that grid cell!
